### PR TITLE
Update openssh version in jupyter image

### PR DESCRIPTION
**Pull request by vicentefb**
The most important fix is to update the OpenSSH package in the Dockerfile. You can usually do this by using the package manager within your Dockerfile (e.g., apt-get update && apt-get install -y openssh-server for Debian/Ubuntu).

### CODE REVIEW

File: modules/jupyter/jupyter_image/notebook_image/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,11 @@
 FROM jupyter/tensorflow-notebook:python-3.10
+USER root
 COPY requirements.txt ./requirements.txt
+RUN apt-get update && apt-get install -y --no-install-recommends openssh-server
+# Create a virtual environment
+RUN python3 -m venv /opt/venv
```
Comment (chiayi) [inline modules/jupyter/jupyter_image/notebook_image/Dockerfile:6]: What benefit does creating a virtual env do here?

Comment (vicentefb) [reply modules/jupyter/jupyter_image/notebook_image/Dockerfile:6]: This approach addresses the following pip warning which was triggered due to `USER root`: 

`Step` #1: WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv`

**Comment by chiayi** [comment]
LGTM!

**Comment by chiayi** [comment]
Discussed offline, v1.1 does not have the the vulnerability that requires updating the openssh version.

**Pull request closed without merging**